### PR TITLE
Credential assignment30

### PR DESCRIPTION
**Pull request by maddiletiboyaness**

### CODE REVIEW

File: force-app/main/default/lwc/credentialAssignment/credentialAssignment.html [hunk 1]
```diff
@@ -1,81 +1,84 @@
...
-              <lightning-layout-item padding="around-small">
-                <div if:true={selectedUserName}>
+    <div class="test3">
+      <div class="test">
+        <div class="slds-grid slds-grid_vertical">
+          <div class="slds-col">
+            <header class="hdr slds-card__header-title slds-truncate">{headerTitle}</header>
+            <span>
+              <lightning-layout>
+                <lightning-layout-item padding="around-small">
```
Comment (nitinsavanurness) [inline force-app/main/default/lwc/credentialAssignment/credentialAssignment.html:9]: Define the sizing for multiple devices

File: force-app/main/default/lwc/credentialAssignment/credentialAssignment.js [hunk 2]
```diff
@@ -77,6 +77,7 @@ export default class CredentialAssignment extends LightningElement {
   @track isDataAvaialable = false;
   @track saveButtonHide = false;
   draftValues = [];
+  headerTitle = 'Credential Assignment';
```
Comment (nitinsavanurness) [inline force-app/main/default/lwc/credentialAssignment/credentialAssignment.js]: Move this to custom label

File: force-app/main/default/lwc/earnedCredentials/earnedCredentials.html [hunk 3]
```diff
@@ -16,7 +16,7 @@
 										<td data-label="Credentials" >
 										
 											<lightning-layout-item class="test4">                                            
-									     <lightning-formatted-rich-text vertical-align="left"  value={u.Badge_Icon__c}></lightning-formatted-rich-text>
+									     <lightning-formatted-rich-text vertical-align="left" style="height: 38px; width: 38px;" value={u.Badge_Icon__c}></lightning-formatted-rich-text>
```
Comment (nitinsavanurness) [inline force-app/main/default/lwc/earnedCredentials/earnedCredentials.html]: Move this styling to css class

**Review by nitinsavanurness — COMMENTED** [review]
@maddiletiboyaness  - Review comments in line, please fix and let me know.

File: force-app/main/default/lwc/earnedCredentials/earnedCredentials.html (hunk 3 above)
Comment (maddiletiboyaness) [reply force-app/main/default/lwc/earnedCredentials/earnedCredentials.html]: Updated code, Please review.

File: force-app/main/default/lwc/credentialAssignment/credentialAssignment.js (hunk 2 above)
Comment (maddiletiboyaness) [reply force-app/main/default/lwc/credentialAssignment/credentialAssignment.js]: Updated code, Please review

**Review by nitinsavanurness — APPROVED** [review]

**Pull request merged**